setup.py

```diff
@@ -24,10 +24,12 @@
      'pyproj',
      'pywavefront',
      'pyyaml',
-     'rdp',
      'scipy',
      'shapely',
+     'alphashape',
      'py3dtiles @ git+https://github.com/VCityTeam/py3dtiles@Tiler',
+     'Pillow',
```

Comment:
The license file should document the licenses of those new dependencies (alphashape, Pillow): refer [here for an example](https://github.com/VCityTeam/3DUSE/blob/master/LICENSE.md#library-dependencies-of-3duse-and-their-associated-respective-licenses).

Use this occasion to fix issue #14